README.md

```diff
@@ -1 +1,27 @@
-# string_reverse_python
\ No newline at end of file
+# String Reverse in Python
+In this assignment, you'll design and implement one of the string manipulation functions that is commonly asked during interviews.
+Remember that a string is an array of characters. Algorithms that worked on restricted arrays will work on strings as well.
```

Comment:
Thinking about it, I don't think the second and third sentences actually apply in Python given that strings are immutable. Is there any other information that would be good to include in this intro section instead?

Reply:
Go ahead and delete the second and third sentences. I like the Python Consideration paragraph you added.